File: Fbtc-Api/Fbtc.Domain/Entities/Isencao.cs
﻿using System;

namespace Fbtc.Domain.Entities
{
    public class Isencao
    {
        public int IsencaoId { get; set; }
        public int? AnuidadeId { get; set; }
        public int? EventoId { get; set; }
        public string Descricao { get; set; }
        public DateTime? DtAta { get; set; }
        public int AnoEvento { get; set; }
        public string TipoIsencao { get; set; }
        public bool Ativo { get; set; }
    }

    public class IsencaoDao
    {
        public int IsencaoId { get; set; }
        public string Descricao { get; set; }
        public int AnoIsencao { get; set; }
        public int QuantIsentos { get; set; }
    }
}

File: fbtc-adm/src/app/components/shared/model/isencao.ts
export class Isencao {
    isencaoId: number;
    anuidadeId: number;
    eventoId: number;
    descricao: string;
    dtAta: Date;
    anoEvento: number;
    tipoIsencao: string;
    ativo: boolean;
}

export class IsencaoDao {
    isencaoId: number;
    descricao: string;
    anoIsencao: number;
    quantIsentos: number;
}
